src/storage/src/hummock/sstable/mod.rs

```diff
@@ -148,24 +78,14 @@ impl DeleteRangeTombstone {
 /// next event key wmk2 (7) (not inclusive).
 /// If there is no range deletes between current event key and next event key, `new_epoch` will be
 /// `HummockEpoch::MAX`.
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
 pub struct MonotonicDeleteEvent {
```

Comment:
Why do we still need to keep `MonotonicDeleteEvent` if we will never use it in the read path?

Reply:
It's used for the backward compatibility to the previous encoding format of `SstableMeta`. If we are sure that all sst meta files written with this range delete event have been compacted in all users' cluster, we can remove this struct and change to return an error or panic when trying to decode it (seeing non-zero length in the list of this event).